### PR TITLE
Fixed the following

### PR DESCRIPTION
**Pull request by limmabean**
-Make batch work with tag search (though kinda dumb)
-Made it so that trackers can click an external link right on the reports or batch page.
-Fixed text overflow on the incidents show page.
-Returned tags to put model but didn't add back the buttons to tag modals.

### CODE REVIEW

**Comment by harshilshah4251** [comment]
When I type in tag and click on search, it automatically shows me the tag id instead of the tag name. If you can fix this, it would be great.
image

- Also, I don't know how to test the tag batch functionality. I searched for a tag and then clicked on grab batch but doesn't work for me. If you can make sure that it works as expected, it would be great. 

Everything else works great.

File: public/angular/templates/incidents/show.html [hunk 1]
```diff
@@ -23,41 +23,43 @@
           </div>
         </div>
         <div class="card-body">
-          <table class="table table-bordered">
+          <table class="table table-bordered incidentShow__table">
```
Comment (harshilshah4251) [inline public/angular/templates/incidents/show.html:26]: is this a naming convention to have __ (double underscores)?

**Comment by harshilshah4251** [comment]
I also added 2 line change for filter by author date since it was easy. You can review that and merge it with this PR.

File: public/angular/templates/incidents/show.html (hunk 1 above)
Comment (limmabean) [reply public/angular/templates/incidents/show.html:26]: Yeah its BEM

**Pull request merged**